### ISSUE TITLE
When Debugging Regular Minecraft Boots Up

### ISSUE THREAD

**Issue by scriptinator** [Wurst-Imperium/Wurst-Client#962]
I have linked the source code to wurst and setup wurst in eclipse, but when I run the main class regular minecraft 1.8 comes up. What should I do?
I am using the Start.java main class

**Comment by Nitrogen-Dioxide** [comment]
You need to install the libs that are on the tutorial, apply the patch file, make a source folder... well you need to setup the workspace.

**Comment by Alexander01998** [comment]
You probably forgot to apply the patch (or didn't apply it correctly).

**Comment by scriptinator** [comment]
Where am I supposed to apply it in the root of the MCP folder?

**Comment by Alexander01998** [comment]
You need to ignore whitespace changes. Do something like this:
```
git apply --ignore-space-change --ignore-whitespace P:\path\to\your\patch\minecraft.patch
```

**Comment by Nitrogen-Dioxide** [comment]
![2015-06-16_11-28-30](https://cloud.githubusercontent.com/assets/11248300/8185475/0e31fda2-141b-11e5-97b4-4425c5b8c90d.png)
![2015-06-16_11-28-54](https://cloud.githubusercontent.com/assets/11248300/8185477/0ec3f6e4-141b-11e5-91bf-f5e728546b34.png)
![2015-06-16_11-29-04](https://cloud.githubusercontent.com/assets/11248300/8185478/0fc80bf2-141b-11e5-8303-979211d229d6.png)
![2015-06-16_11-29-11](https://cloud.githubusercontent.com/assets/11248300/8185480/113333d6-141b-11e5-91a5-92a1ffacf41c.png)

I don't know if it is wrong but I apply the patch this way. And it is showing errors because I already applied it.

**Comment by scriptinator** [comment]
To fix the problem add the official 1.8 jar to the build path
